0000-template.md

```diff
@@ -0,0 +1,49 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- Project: (fill in the related project)
+- Reference Issues: (fill in existing related issues, if any)
+- Implementation PR: (leave this empty)
```

Comment:
Is this bullet ever filled in? When?

Reply:
It's filled once someone makes a PR to implement the changes. This is mostly for historical reasons.